system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/ZookeeperFrameworkSystemTest.java

```diff
@@ -71,9 +71,9 @@ public void testZookeeperFramework_differentPath() throws UnirestException {
         CONTAINER_LIFECYCLE_MANAGEMENT.addAndStart(scheduler);
         ESTasks esTasks = new ESTasks(TEST_CONFIG, scheduler.getIpAddress());
 
-        new TasksResponse(esTasks, CLUSTER.getConfig().getNumberOfSlaves());
+        new TasksResponse(esTasks, 3);
```

Comment:
3 will be difficult to understand. Should be replaced by cluster.getSlaves().length